hive3/build.gradle

```diff
@@ -96,7 +96,7 @@ project(':iceberg-hive3') {
     testImplementation project(path: ':iceberg-hive-metastore', configuration: 'testArtifacts')
     testImplementation project(path: ':iceberg-data', configuration: 'testArtifacts')
 
-    testImplementation("org.apache.avro:avro:1.9.2")
+    testImplementation("org.apache.avro:avro:1.11.1")
```

Comment:
@rdblue do you maybe know why these 2 files define their own versions of libs? I would have assumed that they would use the versions defined in `versions.props`

Reply:
I think the Avro version was somewhat unified across parquet, spark, etc a while ago.

But I think that older Hive versions don't necessarily use the same ones. As these are `testImplementation`, I'm assuming that we normally delegate to the underlying engine but need to specify the right version for the engine during tests.

These might be outdated, but several `testImplementation` versions are overridden here.

Do the tests pass with the updated Avro version specified (or the one from `versions.props`)?

Reply:
tests seem to pass with `1.11.1` here, so I figured we might want to align avro versions. As I can't push to this particular branch here, I opened https://github.com/apache/iceberg/pull/5483